src/meshdb/settings.py

```diff
@@ -36,6 +36,26 @@
 
 USE_X_FORWARDED_HOST = True
 
+MESHDB_ENVIRONMENT = os.environ.get("MESHDB_ENVIRONMENT", "")
```

Comment:
Interesting idea. How will this work for local dev?

Reply:
Defaults to same as before for grand and local